project-base/storefront/utils/toasts/showMessage.tsx

```
@@ -53,22 +60,22 @@ export const showMessage = (message: string, type: 'info' | 'error' | 'success')
             );
         } else {
             toast.error(() => <span dangerouslySetInnerHTML={{ __html: message }} data-tid={TIDs.toast_error} />, {
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)